src/assets/css/librarybrowser.scss

```diff
@@ -840,20 +840,20 @@
     min-width: 0;
     max-width: 100%;
     flex: 1 0 0;
+}
 
-    .layout-mobile & {
-        [dir="ltr"] {
-            @include header-poster-padding;
-        }
+.layout-mobile .infoWrapper {
+    @media all and (max-width: 32em) {
+        position: relative;
+    }
+}
 
-        [dir="rtl"] & {
-            @include header-poster-padding-rtl;
-        }
+.layout-mobile [dir="ltr"] .infoWrapper {
+    @include header-poster-padding;
+}
 
-        @media all and (max-width: 32em) {
-            position: relative;
-        }
-    }
+.layout-mobile [dir="rtl"] .infoWrapper {
+    @include header-poster-padding-rtl;
 }
```

Comment:
This can be rewritten as
```scss
.infoWrapper {
    min-width: 0;
    max-width: 100%;
    flex: 1 0 0;

    .layout-mobile [dir="ltr"] & {
        @include header-poster-padding;
    }

    .layout-mobile [dir="rtl"] & {
        @include header-poster-padding-rtl;
    }

    .layout-mobile & {
        @media all and (max-width: 32em) {
            position: relative;
        }
    }
}
```

Reply:
This doesn't work. I have no idea why.

Reply:
Done.